.github/workflows/ci.yml

```
@@ -24,12 +22,28 @@ on:
 
 jobs:
   build-and-test:
-    name: build and test
+    name: Build & Test ${{ format('({0},{1}){2}', matrix.os, matrix.rust, matrix.experimental && ' - 🦜' || '') }}
```

```suggestion
    name: Build & Test ${{ format('({0}, {1}){2}', matrix.os, matrix.rust, matrix.experimental && ' - 🦜' || '') }}
```
Do you mind if we do that ? It's stressing me out a bit.